src/chainstate/stacks/boot/pox.clar

```diff
@@ -547,10 +557,16 @@
 (define-public (delegate-stack-stx (stacker principal)
                                    (amount-ustx uint)
                                    (pox-addr { version: (buff 1), hashbytes: (buff 20) })
+                                   (start-burn-ht uint)
```

Comment:
Note: I think that this changes in the 2 signatures (+ comments) will outdate https://github.com/blockstack/stacks-blockchain/pull/1957